docs/user_guide/building_blocks/config_objects.md

```diff
@@ -0,0 +1,57 @@
+# Config Objects
+
+Columnflow uses a config file for each analysis, saving the variables specific to the analysis. The
+[order](https://github.com/riga/order) library is used for the conservation and use of the
+metavariables. In this section, the most important config objects to be defined are presented.
+
+## Datasets
+
+TODO
+
+## Processes
+
+TODO
+
+## Shifts
+
+TODO
+
+## "Variable" creation
+
+In order to plot something out of the processed datasets, columnflow uses
+{external+order:py:class}`order.variable.Variable`s. It is therefore not enough to create a new
+column in a {py:class}`~columnflow.production.Producer` for the plotting tasks, it must also be
+translated in a {external+order:py:class}`order.variable.Variable`, which name can be given
+to the argument ```--variables``` for the plotting/histograming task. These
+{external+order:py:class}`order.variable.Variable`s need to be added to the config using the
+function {external+order:py:meth}`order.config.Config.add_variable`. The standard syntax is as
+follows:
```

Comment:
Suggestion for this section with some additional information how Variables are used in histogramming tasks in the next comment

```suggestion
In order to create histograms out of the processed datasets, columnflow uses
{external+order:py:class}`order.variable.Variable`s. These
{external+order:py:class}`order.variable.Variable`s need to be added to the config using the
function {external+order:py:meth}`order.config.Config.add_variable`. The standard syntax is as
follows:
```

Reply:
done